cluster/src/main/java/com/hartwig/pipeline/tertiary/protect/Protect.java

```diff
@@ -0,0 +1,129 @@
+            final ChordOutput chordOutput, final ResourceFiles resourceFiles) {
+        this.purplePurity = new InputDownload(purpleOutput.outputLocations().purityTsv());
+        this.purpleQCFile = new InputDownload(purpleOutput.outputLocations().qcFile());
+        this.purpleDriverCatalog = new InputDownload(purpleOutput.outputLocations().driverCatalog());
+        this.purpleSomaticVariants = new InputDownload(purpleOutput.outputLocations().somaticVcf());
+        this.bachelorTsv = new InputDownload(bachelorOutput.reportableVariants());
+        this.linxFusionTsv = new InputDownload(linxOutput.linxOutputLocations().fusions());
+        this.linxBreakendTsv = new InputDownload(linxOutput.linxOutputLocations().breakends());
+        this.linxDriversTsv = new InputDownload(linxOutput.linxOutputLocations().drivers());
+        this.linxViralInsertionsTsv = new InputDownload(linxOutput.linxOutputLocations().viralInsertions());
+        this.chordPrediction = new InputDownload(chordOutput.predictions());
+        this.resourceFiles = resourceFiles;
+    }
+
+    @Override
+    public String namespace() {
+        return NAMESPACE;
+    }
+
+    @Override
+    public List<BashCommand> inputs() {
+        return List.of(purplePurity,
+                purpleQCFile,
+                purpleDriverCatalog,
+                purpleSomaticVariants,
+                bachelorTsv,
+                linxFusionTsv,
+                linxBreakendTsv,
+                linxDriversTsv,
+                linxViralInsertionsTsv,
+                chordPrediction);
+    }
+
+    @Override
+    public List<BashCommand> commands(final SomaticRunMetadata metadata) {
+        return List.of(new ProtectCommand(metadata.tumor().sampleName(),
+                metadata.tumor().primaryTumorDoids(),
+                VmDirectories.OUTPUT,
+                resourceFiles.actionabilityDir(),
+                resourceFiles.doidJson(),
+                resourceFiles.germlineReporting(),
+                purplePurity.getLocalTargetPath(),
+                purpleQCFile.getLocalTargetPath(),
+                purpleDriverCatalog.getLocalTargetPath(),
+                purpleSomaticVariants.getLocalTargetPath(),
+                bachelorTsv.getLocalTargetPath(),
+                linxFusionTsv.getLocalTargetPath(),
+                linxBreakendTsv.getLocalTargetPath(),
+                linxDriversTsv.getLocalTargetPath(),
+                linxViralInsertionsTsv.getLocalTargetPath(),
+                chordPrediction.getLocalTargetPath()));
+    }
+
+    @Override
+    public VirtualMachineJobDefinition vmDefinition(final BashStartupScript bash, final ResultsDirectory resultsDirectory) {
+        return ImmutableVirtualMachineJobDefinition.builder()
+                .name(namespace())
+                .startupCommand(bash)
+                .namespacedResults(resultsDirectory)
+                .performanceProfile(VirtualMachinePerformanceProfile.custom(4, 12))
+                .build();
```

Comment:
Think we talked some time ago about revising the disk requirements downward for post-alignment steps, not sure if this is a candidate? Looks like a modest VM otherwise so maybe it doesn't take long to run anyway?

Reply:
yeah good idea.